pipeline_steps/common.groovy

```diff
@@ -278,13 +278,54 @@ def archive_artifacts(){
   } finally{
     // still worth trying to archiveArtifacts even if some part of
     // artifact collection failed.
-    archiveArtifacts(
-      allowEmptyArchive: true,
-      artifacts: "artifacts_${env.BUILD_TAG}.tar.bz2,artifact_index.txt"
```

Comment:
#124 removes artifact_index because it isn't useful. Its too large to be a good summary, if you're going to download that, you may as well download the full archive.